src/shared/components/AdminCheckBox.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import Radium from "radium";
+
+import StandardModal from "~/shared/components/StandardModal";
+import CheckBox from "~/shared/components/forms/CheckBox";
+import styles from "~/shared/styles/audience"
+
+const AdminCheckBox = Radium((props) => {
+    return (
+      <div style={props.condition ? styles.adminCheckBox.wrapper : {display: "none"}}>
+          <CheckBox
+              id="modal-checkbox"
+              checked={props.checked}
+              onChange={props.onChange}
+          />
+          <label
+              htmlFor="modal-checkbox"
+          >
+              Admin Only?
+          </label>
+          <p>
+              Selecting this textbox will create the chat with only the admin members of each team.
```

Comment:
It is a checkbox, not a textbox. Also, there should be something like a question mark symbol that can be hovered over and this message should only appear after hovering on the question mark symbol.